### ISSUE TITLE
Image preview, toolbar function unavailable

### ISSUE THREAD

**Issue by umono** [tusen-ai/naive-ui#4223]
### TuSimple/naive-ui version (版本)
2.34.2

### Vue version (Vue 版本)
3.2.45

### Browser and its version (浏览器及其版本)
106.0.5249.119

### System and its version (系统及其版本)
MacOS10.15.3

### Reappearance link (重现链接)
https://codepen.io/umono/pen/GRBgQEw

### Reappearance steps (重现步骤)
```
 <n-image
      width="100"
      src="https://07akioni.oss-cn-beijing.aliyuncs.com/07akioni.jpeg"
 />
```
When clicked, the zoom function on the toolbar cannot be used

### Expected results (期望的结果)
All functions on the toolbar should work normally.

### Actual results (实际的结果)
- Vue warn :Unhandled error during execution of native event handler 
- Uncaught TypeError: Cannot read properties of undefined (reading 'style')

**Comment by jahnli** [comment]
```
    <n-image-group>
      <n-space>
```
缺少对应标签

**Comment by OrbisK** [comment]
should be fixed with #4154
